src/man/sssd-passkey.5.xml

```diff
@@ -0,0 +1,105 @@
+            passkey for
+            <citerefentry>
+                <refentrytitle>sssd</refentrytitle>
+                <manvolnum>8</manvolnum>
+            </citerefentry>.
+            Refer to the <quote>FILE FORMAT</quote> section of the
+            <citerefentry>
+                <refentrytitle>sssd.conf</refentrytitle>
+                <manvolnum>5</manvolnum>
+            </citerefentry> manual page for detailed syntax information.</para>
+    </refsect1>
+
+    <refsect1 id='configuration-options'>
+        <title>CONFIGURATION OPTIONS</title>
+        <para>
+            <variablelist>
+                <varlistentry>
+                    <term>user_verification (boolean)</term>
+                    <listitem>
+                        <para>
+                            Enable or disable the user verification (i.e. PIN,
+                            fingerprint) during authentication.
+                        </para>
+                        <para>
+                            Three different actors come into play when deciding
+                            whether to request user verification: LDAP server,
+                            <citerefentry><refentrytitle>sssd.conf
+                            </refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                            option and the passkey device itself.
+                        </para>
+                        <para>
+                            The configuration option from <filename>sssd.conf
+                            </filename> has higher priority than the LDAP
+                            server and will override them when conflicts occur.
+                        </para>
+                        <para>
+                            In the case of the <filename>sssd.conf</filename>
+                            and the passkey device options the situation is
+                            explained in the following table:
+                        </para>
+                        <informaltable frame='all'>
+                        <tgroup cols='3'>
+                        <colspec colname='c1' align='center'/>
+                        <colspec colname='c2' align='center'/>
+                        <colspec colname='c3' align='center'/>
+
+                        <thead>
+                        <row><entry>sssd.conf</entry><entry>Device</entry>
+                            <entry>Result</entry></row>
+                        </thead>
+                        <tbody>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is configured</entry>
+                            <entry>User verification is requested</entry>
+                        </row>
+                        <row>
+                            <entry>True</entry>
+                            <entry>User verification is not configured</entry>
+                            <entry>Authentication fails</entry>
```

Comment:
Hi,

I think might be more consistent to handle this case as if no (suitable) device is available at the pre-flight check. Then users have the chance to insert another device which requires a PIN before they enter the PIN. 

bye,
Sumit

Reply:
Hi,

no, the pre-flight is is internal and should not be mentioned in the man page explicitly. What I meant is that "Authentication fails" give the impression that authentication fails immediately but I guess in reality what happens in the authentication phase is more a "User verification is requested but it is expected that authentication fails if no other device is inserted in between".

bye,
Sumit

Reply:
Can you take a look at the updated text? I think I have reflected what you mention, but I would prefer to have a confirmation.